FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/FRAuth/FRAuthTests.swift

```diff
@@ -554,4 +554,33 @@ class FRAuthTests: FRAuthBaseTest {
         XCTAssertNil(request?.value(forHTTPHeaderField:"Accept"), "Request should not contain Accept header")
         
     }
+
+
+  @available(iOS 13.0.0, *)
+  func testFRStartWithDiscoveryURL() async throws {
+    do {
+      FRAuthTests.useMockServer = false
+
+      let config =
+      ["forgerock_oauth_client_id":"c12743f9-08e8-4420-a624-71bbb08e9fe1",
+       "forgerock_oauth_redirect_uri": "org.forgerock.demo://oauth2redirect",
+       "forgerock_oauth_scope" : "openid profile email address"]
+      let discoveryURL = "https://auth.pingone.ca/02fb4743-189a-4bc7-9d6c-a919edfe6447/as/.well-known/openid-configuration"
```

Comment:
Are we sure we want to have proper live tenant configurations submitted in the repo? I know it's a pain but in terms of unit tests maybe having mock tests is better and we have integration tests that don't expose the URLs and the tenant details

Reply:
I found a test oidc-configurtion url from gitlab, and changed the code to use that similar to httbin that we are using. Moved the url to FRBaseTest to define in in one place.